AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContextDelegate.h

```diff
@@ -6,13 +6,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol MSAuthTokenContextDelegate <NSObject>
 
+@optional
```

Comment:
Should it be an optional? Any use case that we only implement one method?

Reply:
@jaelim-ms MSPush.m only implements the `didUpdateUserWithAuthToken:` one.